specs/cli/client-add.md

```diff
@@ -0,0 +1,120 @@
+# kiota client add
+
+## Description 
+
+`kiota client add` allows a developer to add a new API client to the `kiota-config.json` file. If no `kiota-config.json` file is found, a new `kiota-config.json` file would be created in thr current working directory. The command will add a new entry to the `clients` section of the `kiota-config.json` file. Once this is done, a local copy of the OpenAPI description is generated and kept in the `.kiota/descriptions` folder.
```

Comment:
what happens if the client is already here?

Reply:
Adding that the command should fail.